src/content/en/ilt/pwa/introduction-to-service-worker.md

```diff
@@ -127,9 +127,27 @@ navigator.serviceWorker.register('/service-worker.js', {
 });
 ```
 
-In this case we are setting the scope of the service worker to `/app/`, which means the service worker will control requests from pages like `/app/`, `/app/lower/` and `/app/lower/lower`, but not from pages like `/app` or `/`, which are higher.
+In this case we are setting the scope of the service worker to `/app/`, which means the service worker will control requests from pages like `/app/`, `/app/lower/` and `/app/lower/lower`, **but not from pages like `/app` or `/`**, which are higher.
+
+If you want the service worker to control higher pages e.g. `/app` (without the trailing slash) you can indeed change the scope option, but you'll also need to set the [Service-Worker-Allowed HTTP Header](https://w3c.github.io/ServiceWorker/#service-worker-script-response) in your server config for the request serving the service worker script, otherwise you'll get an error. Configuration example:
+
+#### main.js
+
+```
+navigator.serviceWorker.register('/service-worker.js', {
```

Comment:
If the location of the service worker script is `/service-worker.js`, then I you'd be able to set the scope to `/app` without a problem, since the service worker script is hosted at the root level. (I tested this locally in Chrome to confirm.)

In order to illustrate a scenario in which setting the scope to `/app` would fail, you'd need to use a service worker script URL hosted at a deeper level, e.g., `/app/service-worker.js`.

(The example in the service worker specification is misleading; I've filed https://github.com/w3c/ServiceWorker/pull/1422)